### PR TITLE
DB import process

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

**Comment by AronNovak** [comment]
image

File: .ddev/config.local.yaml.example [hunk 1]
```diff
@@ -7,6 +7,14 @@ hooks:
         > 0 AND mail LIKE '%~%'"
     # This will be applied to production with a future release.
     - exec: drush sqlq "DELETE FROM system WHERE name='hedley_pusher'"
+    - exec: drush updb -y
+    - exec: drush updb -y
```
Comment (anvmn) [inline .ddev/config.local.yaml.example:11]: @AronNovak really need this twice?

**Review by anvmn — APPROVED** [review]

File: .ddev/config.local.yaml.example [hunk 2]
```diff
@@ -7,6 +7,14 @@ hooks:
...
+    # Modules can come and go, production might be behind.
+    - exec: wget https://ftp.drupal.org/files/projects/registry_rebuild-7.x-2.5.tar.gz
+    - exec: tar -xvf registry_rebuild-7.x-2.5.tar.gz --directory ~/.drush
+    - exec: rm registry_rebuild-7.x-2.5.tar.gz
+    - exec: drush rr
```
Comment (adamhstewart) [inline .ddev/config.local.yaml.example:17]: for me this line failed when it is run before the cache clear.

Comment (adamhstewart) [reply .ddev/config.local.yaml.example:17]: @AronNovak ^^

File: .ddev/config.local.yaml.example (hunk 1 above)
Comment (AronNovak) [reply .ddev/config.local.yaml.example:11]: Yep, not sure why, but locally I saw that after one `updb`, there was still one item to process.

File: .ddev/config.local.yaml.example (hunk 2 above)
Comment (AronNovak) [reply .ddev/config.local.yaml.example:17]: right, adjusted

**Pull request merged**